internal/ci/Dockerfile

```diff
@@ -4,11 +4,6 @@ FROM smartcontract/builder:1.0.1
 ENV DEBIAN_FRONTEND noninteractive
 RUN apt-get update && apt-get install -y jq
 
-# Have to reintroduce ENV vars from Baidu's SGX SDK image
-ENV PATH /root/.cargo/bin:/go/bin:/usr/local/go/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/opt/sgxsdk/bin:/opt/sgxsdk/bin/x64
```

Comment:
Is this related to the environment change? Or just a general cleanup?

I don't see how it relates to the environment variable for `development`

Reply:
Just a cleanup